frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -573,9 +587,28 @@ function ReaderHighlight:_resetHoldTimer(clear)
     end
 end
 
-function ReaderHighlight:onHold(arg, ges)
+function ReaderHighlight:getRegionalZoomCenter(pos)
+    if not self.dimen then
+        self.dimen = self.ui.dimen
+    end
```

Comment:
Since `self.dimen` is only used in this function, it doesn't need to be object's property. You could replace this with
`local dimen = self.ui.dimen`

Reply:
done